starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/Gradle.java

```diff
@@ -92,9 +92,9 @@ public void apply(GeneratorContext generatorContext) {
         )));
 
         generatorContext.addTemplate("gitignore", new RockerTemplate(Template.ROOT, ".gitignore", gitignore.template()));
-        generatorContext.addTemplate("projectProperties", new RockerTemplate("gradle.properties", gradleProperties.template(generatorContext.getBuildProperties().getProperties())));
+        generatorContext.addTemplate("projectProperties", new RockerTemplate(Template.ROOT, "gradle.properties", gradleProperties.template(generatorContext.getBuildProperties().getProperties())));
```

Comment:
Does this need to be at the top level?  None of the properties affect the `infra` module